### ISSUE TITLE
Compatibility with S5 Inverters

### ISSUE THREAD

**Issue by TeamWetLettuce** [Pho3niX90/solis_modbus#62]
This Integration is labelled as S6. Does the Integration work correctly with an S5 Solis inverter? (I have a S5-EH1P6K-L inverter)

**Comment by Pho3niX90** [comment]
> This Integration is labelled as S6. Does the Integration work correctly with an S5 Solis inverter? (I have a S5-EH1P6K-L inverter)

Yes there are a few people using it with S5. It was labeled as S6 since that's the one I am using, but regisrars are the same for all of the solis inverters.
